.github/workflows/make.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - name: Build
         run: make cross
       - name: lint
-        uses: golangci/golangci-lint-action@v3
+        run: make lint
```

Comment:
👍

Reply:
https://github.com/crc-org/admin-helper/pull/50 will take care of keeping up to date both the golangci action and the golangci-lint version we use in `tools` .